support/client/lib/mil-sym/cws.js

```diff
@@ -5370,16 +5513,63 @@ define( function(){
                   "symbolID": "G*FPLCR---****X",
                   "tag": "RESTRICTIVE FIRE LINE",
                   "symbolType": "MultiSegmentLine",
-                  "validModifiers": [ "uniqueDesignation1" ]
+                  "validModifiers": [ "uniqueDesignation1", "uniqueDesignation2", "DTG1", "DTG2" ]
             },
             "TACGRP.MOBSU.SU.FTFDLN": {
                   "symbolID": "G*MPSL----****X",
                   "tag": "FORTIFIED LINE",
                   "symbolType": "MultiSegmentLine"
             }
-       },
+      },
+
+      "offenseAreas": {
+            "TACGRP.C2GM.OFF.ARS.OBJ": {
+                  "symbolID": "G*GPOAO---****X",
+                  "tag": "OBJECTIVE",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1" ]
+            },
+            "TACGRP.C2GM.OFF.ARS.PBX": {
+                  "symbolID": "G*GPOAP---****X",
+                  "tag": "PENETRATION BOX",
+                  "symbolType": "MultiSegmentPolygon"
+            },
+            "TACGRP.C2GM.OFF.ARS.ASTPSN": {
+                  "symbolID": "G*GPOAA---****X",
+                  "tag": "ASSAULT POSITION",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1" ]
+            },
+            "TACGRP.C2GM.OFF.ARS.ATKPSN": {
+                  "symbolID": "G*GPOAK---****X",
+                  "tag": "ATTACK POSITION",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1" ]
+            }
+      },
+
+      "defenseAreas": {
+            "TACGRP.C2GM.DEF.ARS.BTLPSN": {
+                  "symbolID": "G*GPDAB---****X",
+                  "tag": "BATTLE POSITION",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1", "hostile", "echelon" ]
```

Comment:
`echelon` does not appear in the modifiers dialog

Reply:
Yeah, I thought adding the modifier dropdown might be a little too complicated.  It's more of a TODO on the app side.